certutils/gen_cert.go

```diff
@@ -6,8 +6,9 @@ import (
 	"crypto/rand"
 	"crypto/rsa"
 
+	"k8s.io/kubernetes/staging/src/k8s.io/client-go/util/cert"
```

Comment:
This isn't correct. The staging libraries are transferred to their own repos and then consumed from there.